src/lib/maps/Client.ts

```diff
@@ -89,9 +93,21 @@ export class Client {
 
     const parameters = [encodeParameter('api_key', this._credentials.apiKey)];
 
-    if (categories) {
-      const encodedCategories = encodeParameter('categories', categories.toString());
-      parameters.push(encodedCategories);
+    if (dataViewOptions) {
+      Object.entries(dataViewOptions).forEach(([option, value]) => {
+        let optionToEncode;
+
+        if (option === 'bbox') {
+          optionToEncode = (value as number[])?.join(',');
+        } else {
+          optionToEncode = value?.toString();
+        }
+
+        if (optionToEncode) {
+          const encodedOption = encodeParameter(option, optionToEncode);
+          parameters.push(encodedOption);
+        }
+      });
     }
 
     const getUrl = `${url.https}?${parameters.join('&')}`;
```

Comment:
It's not part of this specific PR but we could use a safer method to generate the URL with URLSearchParams or something like that.

Reply:
I didn't know about URLSearchParams! more clear and simpler! :+1: